fmc/src/flow/pcr.rs

```diff
@@ -61,11 +69,63 @@ fn extend_pcr_common(env: &mut FmcEnv, hand_off: &HandOff, pcr_id: PcrId) -> Cal
     let rt_tci: [u8; 48] = okref(&rt_tci)?.into();
     env.pcr_bank.extend_pcr(pcr_id, &mut env.sha384, &rt_tci)?;
 
```

Comment:
`extend_journey_pcr` is only invoked if the reset reason is cold-reset or update-reset. Do we have a way to trigger a reset whose reason is not either of those, so we can test that logic?